modules/apps/29-fee/ibc_middleware.go

```diff
@@ -168,6 +168,14 @@ func (im IBCMiddleware) OnChanCloseConfirm(
 		return err
 	}
 
+	if im.keeper.IsLocked(ctx) {
+		return types.ErrFeeModuleLocked
+	}
```

Comment:
was there a reason this change wasn't applied to `OnChanCloseInit` as well? Will this be done in a followup?